@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -886,11 +906,10 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
             const modifiedAlpha = Math.abs(alpha - cameraPitch);
 
             // Use tangent based far plane if horizon is within field of view
-            const farTangent = this.getTangentBasedFarPlane(camera, d, r, modifiedAlpha);
-            farPlane =
-                halfFovAngle >= modifiedAlpha
-                    ? farTangent
-                    : this.getTiltedFovBasedFarPlane(d, r, halfFovAngle, cameraPitch);
+            const horizonWithinView = halfDiagonalFovAngle >= modifiedAlpha;
```

Comment:
This is not right anymore. Before, since we were testing the visibility of the tangent point at the top of the view, alpha (the angle from normal to tangent) was reduced by the camera pitch, which is a vertical angle. Now that we test the visibility of a tangent point in the top right view corner, we can't substract the camera pitch angle just like that, since it's not in the same direction as the angle to the new tangent point.

Reply:
Thanks, I will fix.